import_export/README.md

```diff
@@ -2,18 +2,29 @@
 
 ## Overview
 
-This folder includes the code for importing data from and exporting data to various external data sources where original documents and translated content is hosted. Currently, Google documents and Wordpress sites are the two data sources integrated into the ELSA continuous localization pipeline.

```

Comment:
Why delete this sentence?

Reply:
I was removing the wordpress stuff from this readme but forgot this module includes both google docs and wordpress so will add back